src/styles/index.scss

```diff
@@ -1,3 +1,111 @@
+$color-main-light: #fff;
+$color-main-dark: #060b35;
+$color-secondary: #616070;
+$color-button: #00acdc;
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  color: $color-main-dark;
+}
+
 body {
   margin: 0;
 }
+
+p,
+h3 {
+  margin: 0;
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background-color: $color-main-light;
+  border: 1px solid #F3F3F3;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    color: $color-secondary;
+    margin-bottom: 16px;
+  }
+
+  &__reviews-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    color: $color-secondary;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    height: 40px;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    line-height: 40px;
+    color: $color-main-light;
+    box-sizing: border-box;
+    background-color: $color-button;
+    border-radius: 5px;
+    border: none;
+
+    &:hover {
+      background-color: transparent;
+      color: $color-button;
+      border: 1px solid $color-button;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
```

Comment:
Move it to a separate file.
![image](https://github.com/mate-academy/layout_product-cards/assets/64101898/d07763d1-a374-477b-a440-ab7cde3e791d)